### PR TITLE
Challenge 2 - Capitalize Strings

### PR DESCRIPTION
**Pull request by Tbones41**

### CODE REVIEW

File: Challenge 2.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def first_letter_capitalise():
+    test_sentence = str(input("Your Sentence here >>"))
+    print(test_sentence.title())
+
+
+first_letter_capitalise()
```
Comment (Del-sama) [inline Challenge 2.py]: See last challenge's comment about leaving an extra line. Consider installing a linter into your editor. 
Let me know if you need help with that

Comment (Del-sama) [reply Challenge 2.py]: "string".capitalize() would have been a better approach. See `https://elearning.wsldp.com/python3/python-capitalize-string/`

Comment (Del-sama) [reply Challenge 2.py]: "string".capitalize would have been a better option. See `https://elearning.wsldp.com/python3/python-capitalize-string/`

Comment (Del-sama) [reply Challenge 2.py]: > "string".capitalize would have been a better option. See `https://elearning.wsldp.com/python3/python-capitalize-string/`

Comment (Tbones41) [reply Challenge 2.py]: True. I originally used `.capitalize` in my first draft of the code. However, I noticed that it capitalizes only the first letter of the first word of a string while `.title` capitalizes the first letter of every word in the string. The latter seemed more in line with the challenge.
Also, if I were to make use of `.capitalize`, I'd need to split the string into parts of a list, capitalize them then output it. I chose the shorter code.

Comment (Del-sama) [reply Challenge 2.py]: You're right, I missed that.

Comment (Del-sama) [reply Challenge 2.py]: Change the print statement to `return test_sentence.title()`.  See `https://stackoverflow.com/questions/7129285/what-is-the-purpose-of-the-return-statement`

**Pull request merged**